python/dgl/geometry/mxnet/fps.py

```diff
@@ -24,13 +24,16 @@ def __init__(self, npoints):
         super(FarthestPointSampler, self).__init__()
         self.npoints = npoints
 
-    def forward(self, pos):
+    def forward(self, pos, random_start=True):
```

Comment:
Can we change it to `start_idx`, which appoint the index of the starting point if is not `None`?